package.json

```diff
@@ -32,7 +32,7 @@
     "test:group:broker": "yarn test:local --testPathPattern 'src/broker/.*'",
     "test:group:admin": "yarn test:local --testPathPattern 'src/admin/.*'",
     "test:group:producer": "yarn test:local --testPathPattern 'src/producer/.*'",
-    "test:group:consumer": "yarn test:local --testPathPattern 'src/consumer/.*'",
+    "test:group:consumer": "yarn test:local --testPathPattern 'src/consumer/.*.spec.js'",
```

Comment:
This shouldn't be necessary 🤔 Did you see some issue?

Reply:
@Nevon Yes, I added three fixtures defined as `.js` files, which caused jest to complain about files with no exported tests. Normally I'd define the fixtures as JSON but these objects have some values that need to be defined as buffers, which AFAIK would have been a pain in JSON. So I updated the test pattern to only look for `spec.js` files (which is the convention we use anyways).